src/components/social/google/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import GoogleLogin from 'react-google-login';
+
+import LoginGoogle from '../../../actions/socialLogin/GoogleActions';
+import store from '../../../store';
+
+class GoogleOption extends Component {
+  responseGoogle = (response) => {
+    const GoogleToken = response.tokenId;
+    store.dispatch(LoginGoogle(GoogleToken));
+    document.location.href = '/successpage';
+  };
+
+  render() {
+    return (
+      <div className="social-apps">
+        <GoogleLogin
+          clientId="946814017153-bl2ajac6l961e2it5kf1vm5hskj7nih7.apps.googleusercontent.com"
```

Comment:
@Rhytah, consider setting this in the environmental variables.

Reply:
@Emmanuel-Dominic I have done this.